pkgs/applications/networking/mailreaders/thunderbird/default.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
 
   src = fetchurl {
     url = "ftp://ftp.mozilla.org/pub/thunderbird/releases/${verName}/source/thunderbird-${verName}.source.tar.bz2";
```

Comment:
I guess you will have to update this one, base on ftp://ftp.mozilla.org/README .

```
The contents of ftp://ftp.mozilla.org has moved to http://archive.mozilla.org
```